serve/package.go

```diff
@@ -132,6 +132,9 @@ func (s *PluginServe) build(pluginDirectory string, target plugin.BuildTarget, d
 	args := []string{"build", "-o", pluginPath}
 	args = append(args, "-buildmode=exe")
 	args = append(args, "-ldflags", ldFlags)
+	// This disables gRPC tracing, which introduces a dependency that blocks dead code elimination\
```

Comment:
This conflicts with the FIPS support I believe as it will override https://github.com/cloudquery/cloudquery-private/pull/7320/files#diff-e6ce7bc316a2e750a956a86aa73a9d0e59192339395eb4c0c1a70d5ff19a6df1R93

cc @przste-go.

Maybe we need to add an option to the SDK to pass build tags

Reply:
Would be good to do the build flags as an option so we can gradually rollout the `grpcnotrace` change as well

Reply:
[07a9db1](https://github.com/cloudquery/plugin-sdk/pull/2129/commits/07a9db138b42d8579e910d2eb474602dc7a6aef1)